src/App/App.test.tsx

```diff
@@ -10,42 +10,55 @@ import App from './App';
 
 jest.mock('../apiCalls')
 
-const mockGenres: string[] = [
-  'new orleans brass',
-  'nerdcore',
-  'kindermotown',
-  'military western ska'
-]
 
-const mockPlaylist1: Playlist = {
-  id: 1,
-  name: 'military western ska',
-  isSaved: false,
-  tracks: [
-    {
-      "mbid": "e2bad905-75a4-499d-bdea-2e916d73ad76",
-      "artist": {
-        "name": "Sublime",
-        "artistUrl": "https://www.last.fm/music/Sublime"
-      },
-      "duration": "208",
-      "songName": "Smoke Two",
-      "songUrl": "https://www.last.fm/music/Sublime/_/Smoke+Two+Joints"
-    },
-    {
-      "mbid": "86906b91-fdbd-4407-bcf1-6105f42525e5",
-      "artist": {
-        "name": "Pat Metheny Group",
-        "artistUrl": "https://www.last.fm/music/Pat+Metheny+Group"
-      },
-      "duration": "167",
-      "songName": "Forward March",
-      "songUrl": "https://www.last.fm/music/Pat+Metheny+Group/_/Forward+March"
-    }
-  ]
-}
 
 describe('App', () => {
+  let mockGenres: string[], mockPlaylist1: Playlist, mockPlaylist2: Playlist
+  beforeEach(() => {
+    mockGenres= [
+      'new orleans brass',
+      'nerdcore',
+      'kindermotown',
```

Comment:
Great catch realizing that, as global variables, the data was storing like it had already been clicked instead of resetting from the beginning. Using beforeEach definitely made it easier to solve that last test.